### PR TITLE
Add Status Box Component

### PR DESCRIPTION
**Pull request by CannonLock**

### CODE REVIEW

**Comment by CannonLock** [comment]
Failure is due to the fact that I built this off of the graph addition then separated it after. Once that gets merged things will work.

File: origin_ui/src/app/(dashboard)/page.tsx
Comment (Techming) [inline origin_ui/src/app/(dashboard)/page.tsx:39]: * It can be very helpful if we can display the error messages for the the critical and warning status.
* We might want to have different colors for the different status (critical/warning)
* The color fill is a bit heavy for the user reading the dashboard for OK and warning status. A colored-border box might be more user friendly?
* Capitalize the first letter of the status?

**Review by Techming — APPROVED** [review]
LGTM. Let some style/color comments but shouldn't block this PR to be merged. We can address the color coding in a follow-up issue if necessary. Approved.

**Pull request merged**
